### PR TITLE
Added the new bot.js file and the response.js files

### PR DESCRIPTION
**Pull request by zolwethu**
Added a new file

### CODE REVIEW

File: Nova/control/chatbot-services.js [hunk 1]
```diff
@@ -158,10 +164,40 @@ const fetchResponse = (userInput) => {
...
+                botMessage = "See ya, wouldn´t wanna be ya!"
+
+            } else if (userInput.toLowerCase().includes("cheer")) {
+                botMessage = "Oh cheers love!"
+
+            } else if (userInput.toLowerCase().includes("xoxo")) {
+                botMessage = "oh I like this one! Bye naughty"
+
             } else {
                 botMessage = "What would you like to know about Sibusiso? Please choose ";
```
Comment (TshepoMagagula) [inline Nova/control/chatbot-services.js:202]: This message keeps popping up but there is no options to choose from

File: Nova/control/chatbot-services.js [hunk 2]
```diff
@@ -36,35 +36,41 @@ const fetchResponse = (userInput) => {
             // GREETINGS
             if (userInput.toLowerCase().includes("hello")) {
```
Comment (TshepoMagagula) [inline Nova/control/chatbot-services.js:37]: Try to make the chat bot control the conversation, instead of saying "type anything"

**Comment by BAHLEMAJOLA** [comment]
image
@zolwethu Please fix the spacing between the chatbot's welcome text and the user's message.

**Review by BAHLEMAJOLA — CHANGES_REQUESTED** [review]
image
@zolwethu Please fix the spacing between the chatbot's welcome text and the user's message.

**Comment by TshepoMagagula** [comment]
> image @zolwethu Please fix the spacing between the chatbot's welcome text and the user's message.

**Pull request closed without merging**